src/tests/record.rs

```diff
@@ -275,6 +276,10 @@ fn replay_http(
     mut exchange: Exchange,
     stdout: &mut Write,
 ) -> Box<Future<Item = hyper::Response<hyper::Body>, Error = hyper::Error> + Send> {
+    fn check_ignored_request_header(name: &str) -> bool {
```

Comment:
What do you think about making this a static vec instead, and calling `IGNORED_HEADERS.contains`?